docs/flex/interactive/data_import.md

```diff
@@ -6,6 +6,28 @@ In our guide on [using custom graph data](./custom_graph_data.md), we introduced
 
 Currently we only support import data to graph from local `csv` files or `odps` table. See configuration `loading_config.data_source.scheme`.
 
+## Column mapping
+
+When importing vertex and edge data into a graph, users must define how the raw data maps to the graph's schema. 
+This can be done using a YAML configuration, as shown:
+
+```yaml
+- column:
+    index: 0  # Column index in the data source
+    name: col_name # If a column name is present
+  property: property_name  # The mapped property name
+```
+
+The column mapping requirements differ based on the data source:
+
+#### Import from CSV 
+
+You can provide either `index`, `name`, or both.
```

Comment:
which one has higher priority if `index` and `name` exist simultaneously ?

Reply:
Updated in documentation.
If Both name and index are specified, we will check whether they points to the same column.

Reply:
What if they are not?

Reply:
If not, exception will be returned.